docs/index.md

```diff
@@ -142,6 +143,47 @@ Then run tests with:
 $ npm test
 ```
 
+## Run Cycle Overview
+
+A brief outline on the order Mocha's components are executed.
+Worth noting that all hooks, `describe` and `it` callbacks are run in the order they are defined (i.e. found in the file).
+
+``` java
```

Comment:
Why is the codeblock language set to 'java'? Maybe use 'text' instead?

Reply:
Oh, I see. You were highlighting the JS functions in red.
How about a comment to that effect?
```markdown

[//]: # (Used 'java' language so Mocha globals are highlighted)
```

Reply:
Nice, done.

Reply:
For the comment to function correctly, it must be on its own line. Think it also requires surrounding blank lines (both before and after it).

Reply:
Also note now that the highlighting you had from before no longer occurs. Only dashed lines are shown in red (as though you had used 'text' for codefence language).

Did quick test and defining the codefence language as `js` seems to come closest to your original, highlighting almost all Mocha globals (`after` doesn't for some odd reason). Grrrr.
Markdown highlighting hacks - gotta luv 'em, eh?

Reply:
You didn't make any changes here. Surprised `markdownlint` isn't b*tching about the comment.

But does the lack of highlighting eliminate the need for the comment?

Reply:
codefence highlighting/markdown comment is what has been blocking it for 3 weeks...

Reply:
Had missed this comment, will address it.

Reply:
@plroebuck tried myself agree `js` seemed fine and as its more preferable used `js` and removed the comment. If this is not right please instruct. Thank you.